.github/labels.yml

```diff
@@ -42,23 +42,23 @@
   description: "No further development until this is fixed"
   color: C0392B
 
-- name: Critical Priority
+- name: critical priority
```

Comment:
I wouldn't use spaces, but instead dashes, backslash...

Reply:
I understand but it looks very weird having ``DO-NOT-MERGE``, ``won't-fix``, etc.. I will keep the advice in my mind though.